### DISCUSSION TITLE
Make the horizontal line optional

### DISCUSSION THREAD

**Discussion by ronrdev** [vslinko/obsidian-zoom#70] [Ideas]
I can't figure out what this horizontal line is/where it comes from. I am trying to remove it, but can't identify it with the inspector. Is it just broken on dark theme? Is it an alignment gap? Is is a border of some element? Is it hard coded somehow? Add option to style. In dark mode it is distracting. I'm using minimal theme.

![Screenshot_2022-05-02_02-25-16](https://user-images.githubusercontent.com/13022919/166193840-f878c827-40ae-419e-a532-29e435009cdb.png)
![Screenshot_2022-05-02_02-26-32](https://user-images.githubusercontent.com/13022919/166193847-0f1dd0a1-4dad-4576-ac46-c51a381a6ded.png)

**Comment by arielmang** [comment]
I'm having the same issue, if you have figured out a way to hide it could you share? Thank you!

**Comment by arielmang** [comment]
Just found a way to hide it, idk if it's correct or not:

`.ͼ2 .cm-panels-top {
    border-bottom: 1px !important;
}`

**Comment by ronrdev** [comment]
I applied the css, and it is working great. At least visually. Thank you for sharing your solution!
